src/librustc_mir/borrow_check/error_reporting.rs

```diff
@@ -690,9 +690,10 @@ impl<'cx, 'gcx, 'tcx> MirBorrowckCtxt<'cx, 'gcx, 'tcx> {
         if let Some(decl) = local_decl {
             if let Some(name) = decl.name {
                 if decl.can_be_made_mutable() {
-                    err.span_label(
+                    err.span_suggestion(
                         decl.source_info.span,
-                        format!("consider changing this to `mut {}`", name),
+                        "consider changing this to",
+                        format!("mut {}", name),
                     );
```

Comment:
Use `span_suggestion_with_applicability` and `Applicability::MachineApplicable`, so that `rustfix` and VScode can apply this automagically.